yb-voyager/cmd/importData.go

```diff
@@ -258,27 +259,35 @@ func importData(importFileTasks []*ImportFileTask) {
 	if !dbzm.IsDebeziumForDataExport(exportDir) {
 		executePostImportDataSqls()
 	} else {
+		status, err := dbzm.ReadExportStatus(filepath.Join(exportDir, "data", "export_status.json"))
+		if err != nil {
+			utils.ErrExit("failed to read export status for restore sequences: %s", err)
+		}
 		if changeStreamingIsEnabled(importType) {
 			color.Blue("streaming changes to target DB...")
 			err = streamChanges()
 			if err != nil {
 				utils.ErrExit("Failed to stream changes from source DB: %s", err)
 			}
-		}
 
-		// in case of live migration sequences are restored after cutover
-		// otherwise for snapshot migration, directly restore sequences
-		status, err := dbzm.ReadExportStatus(filepath.Join(exportDir, "data", "export_status.json"))
-		if err != nil {
-			utils.ErrExit("failed to read export status for restore sequences: %s", err)
-		}
-		err = tdb.RestoreSequences(status.Sequences)
+			// in case of live migration sequences are restored after cutover
+			err = tdb.RestoreSequences(status.Sequences)
```

Comment:
Isn't this wrong? This change is restoring sequence values as they were immediately after the snapshot export. What if the sequence values were changed during the streaming phase?

Reply:
yeah right, nice catch.

Reply:
moving it back inside the `if-else` clauses